trussworks/react-uswds
# build(deps): Update storybook to 6.2.x

# Summary

Dependabot + Kodiak were unable to update this dependency automagically due to Happo timing out on comparisons for this storybook version, so this PR serves to make the required changes to get passed the hiccups.

Edit: Well, its just... working? Maybe it was the update to Happo storybook plugin (though the patch notes for it didn't seem to indicate anything of the sort, I just did it while I was here)

## Related Issues or PRs

closes: https://github.com/trussworks/react-uswds/issues/1109

## How To Test

The main bug here was Happo not running diffs for this updated version of storybook. Since this is a change to storybook, it is also important to make sure storybook is still working. 

- [x] test storybook locally
- [x] test storybook CI deployment
- [x] verify Happo diff does not time out and generally looks good.
